scripts/LedSignMock.py

```diff
@@ -0,0 +1,36 @@
+
+import curses
+import argparse
+
+parser = argparse.ArgumentParser(description="Prints mock output.")
+parser.add_argument('--mock-output', help = "Displays Mock Output", )
+args = parser.parse_args()
+
+menu = ['   ','SCE LED Sign Visualization ', ' ','Most Recent Health Check: ', 'Most Recent Sign Update: ',' ',
+'Current Sign Text:', 'Current Brightness:', 'Current Scroll Speed:', 'Current Text Color: ', 'Current Background:',
+'Current Border Color: ' ]
+
+class LedSignMock(data):
+    def print_menu(stdscr, selected_row_idx):
+        data = "info here"
+        stdscr.clear()
+        stdscr.border(0)
+        h, w = stdscr.getmaxyx()
+        for idx, row in enumerate(menu):
+            x = w//3 - len(menu)
+            y = h//3 - len(menu)//3 + idx
+            stdscr.addstr(y, x, row)
+        stdscr.refresh()
+
+    def main(stdscr):
+        curses.curs_set(0)
+        current_row = 0
+        print_menu(stdscr, current_row)
+        x = stdscr.getch()
+        while x != 27:
+            x = stdscr.getch()
+        curses.endwin()
+        exit()
+        print_menu(stdscr, current_row)
+
+    curses.wrapper(main)
```

Comment:
try putting all of the code in this file to be within the class. you can then use the imported class in `led_sign_server.py` by creating an object and then invoking the visual like:

```py
from scripts import LedSignMock 

# later on in some function
mock = LedSignMock(self.sign_data)
# this will run the code to create the visual
mock.create_visual()
```